### PR TITLE
Web app: open in editor copy updates

### PR DESCRIPTION
**Pull request by camdencheek**
Some updates to the editor button copy to make it more obvious what is meant by the "Default projects path"

[Requesting Slack thread](https://sourcegraph.slack.com/archives/C03CSAER9LK/p1710333723801189)

## Test plan

Manually tested 

CleanShot 2024-03-13 at 11 46 57@2x

### CODE REVIEW

**Review by bahrmichael — APPROVED** [review]

**Review by BolajiOlajide — APPROVED** [review]

File: client/web/src/open-in-editor/OpenInEditorPopover.tsx [hunk 1]
```diff
@@ -95,6 +95,11 @@ export const OpenInEditorPopover: React.FunctionComponent<
                         onChange={onProjectPathChange}
                         className={classNames('mr-sm-2')}
                     />
+                    <aside className="small text-muted">
+                        The directory that contains your repository checkouts. e.g. if this repository is checked out to{' '}
```
Comment (sqs) [inline client/web/src/open-in-editor/OpenInEditorPopover.tsx:99]:
```suggestion
                        The directory that contains your repository checkouts. For example, if this repository is checked out to{' '}
```

File: client/web/src/open-in-editor/OpenInEditorPopover.tsx [hunk 2]
```diff
@@ -95,6 +95,11 @@ export const OpenInEditorPopover: React.FunctionComponent<
...
+                        <code>/Users/username/projects/cody</code> then set your default projects path to{' '}
```
Comment (sqs) [inline client/web/src/open-in-editor/OpenInEditorPopover.tsx:100]:
```suggestion
                        <code>/Users/username/projects/cody</code>, then set your default projects path to{' '}
```

**Review by sqs — APPROVED** [review]
thanks!

**Review by toolmantim — APPROVED** [review]

**Review by taiyab — APPROVED** [review]
LGTM! 👍

**Pull request merged**